pkg/controller/dataimportcron-controller_test.go

```diff
@@ -439,7 +439,7 @@ var _ = Describe("All DataImportCron Tests", func() {
 				Expect(err).ToNot(HaveOccurred())
 				ts := pvc.Annotations[AnnLastUseTime]
 				Expect(ts).ToNot(BeEmpty())
-				Expect(ts > lastTs).To(BeTrue())
+				Expect(strings.Compare(ts, lastTs)).Should(Equal(1))
```

Comment:
is it better?

Reply:
Currently, there is a bug in ginkgolinter (solved in v0.11.2) that replaces
```go
Expect(ts > lastTs).To(BeTrue())
```
with
```go
Expect(ts).To(BeNumerically(">", lastTs))
```

So I want to avoid false positive here.

In addition, as far as I understand, the best practice for lexicographic comparisons in go, is using  `strings.Compare()`.